README.md

```diff
@@ -1,2 +1,65 @@
-# Covid Tracker App
- 
+# CryptoVerse Mobile App
+![cryptobanner](https://user-images.githubusercontent.com/5249414/181251730-6a6dc577-6d54-42f6-8e07-3ac437dbe886.png)
+
+
+> The application will allow users to filter a list of crypto assets and view detail. If you are a crypto lover, you can use this app to stay updated on current prices and market cap.
+
+## Presentation
+[Click here to watch my presentation](https://www.loom.com/share/ecfa74659b8e441781ca0378fd4337b4) | 
+[Click here for live demo](https://62e1463c4c84cd27a8ed2f61--ephemeral-jalebi-c9b606.netlify.app/)
+
+
+## Built With
+
+- JavaScript, HTML, CSS
+- React.js
+- Redux, Redux-toolkit
+- Jest
+
+## Getting Started
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Prerequisites
+
+- Node
+- Git
+- Code Editor
+
+## Insall
+
+Run the following command to get a working version locally
+
+- `git clone https://github.com/kwekubright/CryptoVerse-Mobile-App.git`
+- `cd cryptoverse-mobile-app`
+- `npm install`
+- `npm start`
+
+## Test
+
+Run this command to test the application
+
+`npm test`
+
+## Authors
+
+Bright Ahiadeke
+
+- GitHub: [@kwekubright](https://github.com/kwekubright)
+- Twitter: [@kwekubright_](https://twitter.com/kwekubright_)
+- LinkedIn: [Bright ahiadeke](https://www.linkedin.com/in/kwekubright/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] Kindly give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in your acknowledgments section.